app/assets/stylesheets/front/pages/meditations.sass

```diff
@@ -1,132 +1,77 @@
 @import "../../mixins/vars"
 
 body.meditations-index
-
   &.open-menu
     @include tablet-or-smaller
       overflow: hidden
       height: 100vh
 
-  main section
-    padding: 20px 0
-    margin-bottom: 0
-    
-    &.page-title
-      padding: 25px 0
-      
+  section.intro
+    margin: 0
+
     .title
-      text-align: center
-      font-size: 2.125em
+      margin-top: 1em
+      font-size: 2.5em
       font-family: $raleway-bold
-      margin: 0
-      letter-spacing: .05em
-
-      @include desktop
-        font-size: 2.8125em
-        margin: 24px 0
+      line-height: 150px
 
-      @include tablet-or-smaller
-        font-size: 1.675em
+      display: flex
+      align-items: center
+      justify-content: center
 
-      @include mobile
-        line-height: 1.25em
-        
-    h2.title
-      font-family: $raleway-medium
-      font-weight: 500
-      letter-spacing: .02em
-      margin: 35px 0 25px
+      &:after
+        width: 100%
+        left: auto
+        right: 50%
 
-      @include desktop
-        margin: 35px 0 50px
+  section.featured-meditations
+    padding: 20px 0
+    margin-bottom: 0
 
-  section.meditation-video
     @include desktop
       padding-top: 50px
 
-  section.intro
-    margin-bottom: 50px
+    .container
+      .meditation-item
+        display: inline-block
+        width: 33.33%
+        padding: 0 15px
 
-  nav.filters
-    margin-bottom: 2.8em
+  section.custom-meditation
+    padding: 20px 0
+    margin-bottom: 0
 
-    @include mobile
+    h2.title
       text-align: center
+      font-size: 2.125em
+      font-family: $raleway-medium
+      letter-spacing: 0.02em
+      margin: 35px 0 25px
 
-    .filter.button
-      margin-right: 44px
-      margin-bottom: 0.625em
+      @include desktop
+        font-size: 2.8125em
+        margin: 35px 0 50px
+
+      @include tablet-or-smaller
+        font-size: 1.675em
 
       @include mobile
-        margin: 10px 22px
+        line-height: 1.25em
 
-  .sidetext
-    display: none
-    @include desktop
-      display: block
+    .sidetext
+      @include laptop-or-smaller
+        display: none
 
   #meditations-triangle::after
     bottom: auto
     right: 10.5%
     top: -120px
     zoom: 0.875
 
-  .grid-row
-    display: flex
-    margin: 0 -15px
-    &.slick-slider
-      margin: 0
-      > *
-        padding: 0
-
-    > *
-      padding: 0 15px
-
-    .grid-sizer
-      width: 33.33%
```

Comment:
All class names or ids that start with `grid` are related to `Grid.js` we need to use different naming.

This styling should also be scoped to the `section.featured-meditations` section, so I replaced it with a `.container` above.